frontend/tailwind.config.js

```diff
@@ -130,22 +108,15 @@ module.exports = {
       ],
     },
     fontSize: {
-      '13': '13px',
-      '14': '14px',
-      '15': '15px',
-      '16': '16px',
-      '17': '17px',
-      '18': '18px',
-      '20': '20px',
-      '22': '22px',
-      '24': '24px',
-      '30': '30px',
-      '35': '35px',
-      '40': '40px',
-      '50': '50px',
-      '62': '62px',
-      '66': '66px',
-      '70': '70px',
+      'base': '10px', 
+      'lg-base': '12px', 
+      'body': '16px',
+      'small': '14px',
+      'large': '18px',
+      'large-1': '4.7rem',
+      'large-2': '3.2rem',
+      'large-3': '2.4rem',
+      'large-4': '1.8rem'
```

Comment:
you know that rem unit is changes depending on the body font size? and it is locked to 16px

Reply:
It changes based on the font-size of the <html> element. That's the idea: but changing <html>'s font-size at a larger bp you change all the other sizes on the page which ought to be updated.

Reply:
@adamf321 `rem` is only relative to the html (root) font-size, while `em` is relative to the font-size of its direct or nearest parent (which is what I think you refer to). I didn't understand the second part of your comment: "That's the idea: but changing 's font-size at a larger bp you change all the other sizes on the page which ought to be updated."

Reply:
I'm saying this block of CSS will control all the font sizes on the page:
```
html {
    @apply text-base text-font-primary;
}

@screen lg {
    html {
        @apply text-lg-base;
    }
}
```

On mobile <html> size is 10px so large-1 will compute to 47px (4.7rem)
On desktop <html> size is 12px so large-1 will compute to 56px (still 4.7rem)

Reply:
That's right, makes sense.